cg/meta/store/base.py

```diff
@@ -0,0 +1,52 @@
+""" Base module for building MIP bundles for linking in Housekeeper"""
+import datetime as dt
+
+from cg.exc import AnalysisDuplicationError, BundleAlreadyAddedError
+
+
+def add_new_analysis(bundle_data, case_obj, status, version_obj):
+    """Function to create and return a new analysis database record"""
+    pipeline = case_obj.links[0].sample.data_analysis
+    pipeline = pipeline if pipeline else "mip-rna"
```

Comment:
If this is in base.py I guess there is no sane default here and pipeline should be required to be defined at this point otherwise we do not know where the data is coming from and should say so and croak

Reply:
Added exception to catch missing `data_analysis' fields.